client/src/components/Header.tsx

```diff
@@ -74,7 +74,7 @@ export function Header({ title, courseName, username }: Props) {
         }}
       >
         <Space size={24}>
-          <Link href="/">
+          <Link href="/" legacyBehavior>
             <a>
               <img
                 style={{ height: 30 }}
```

Comment:
```
          <Link href="/">
            <img
              style={{ height: 30 }}
              className="header-logo"
              src="/static/images/logo-rsschool3.png"
              alt="Rolling Scopes School Logo"
            />
          </Link>
```
do you really need the legacyBehavior flag?
I tried it without it and it seems to work